dotcom-rendering/package.json

```diff
@@ -258,8 +259,9 @@
 		"trusted-types": "2.0.0",
 		"ts-loader": "9.4.3",
 		"ts-unused-exports": "8.0.5",
-		"tslib": "2.5.0",
-		"typescript-json-schema": "0.54.0",
+		"tslib": "2.5.3",
+		"type-fest": "3.12.0",
```

Comment:
Where is this used?

Reply:
<img width="1198" alt="Screenshot 2023-07-05 at 15 09 48" src="https://github.com/guardian/dotcom-rendering/assets/77005274/1fe21c71-056d-4a91-b6f9-33f0301bb762">